app/controllers/collaborators_controller.rb

```diff
@@ -0,0 +1,53 @@
+class CollaboratorsController < ApplicationController
+  before_action :authenticate_user!
+
+  def new
+    @wiki = Wiki.find(params[:wiki_id])
+    @collaborator = @wiki.collaborators.new
```

Comment:
So it is not necessarily bad practice but what I would do it this way
```
@collaborators = Wiki.find(@wiki).collaborators
```
That is a little more Rails-like